static/locales/en.json

```diff
@@ -284,5 +284,8 @@
   "Connecting...": "Connecting...",
   "Downloading stream... not long left now!": "Downloading stream... not long left now!",
   "Downloading: ": "Downloading: ",
-  "% complete": "% complete"
-}
\ No newline at end of file
+  "% complete": "% complete",
```

Comment:
I see `comma-dangle` in _.eslintrc.json_ and _.prettierrc.json_, but doesn't look like it's being respected.